main/zoe/api/zoe-contract-facet.md

```diff
@@ -127,7 +127,8 @@ mySynchronousMint.mintGains({ MyKeyword: amount }, seat);
 Zoe has a single `invitationIssuer` for the entirety of its
 lifetime. This method returns the Zoe `InvitationIssuer`, which
 validates user-received `invitations` to participate
-in contract instances.
+in contract instances. It has a one-to-one association with a
```

Comment:
This may be my personal quirk, but it seems wrong to me to use the words "one-to-one" when the relationship in one direction is "one-to-many". There are many invitations per issuer and per mint, so I think one-to-one gives the wrong connotations. Instead, we can say it shares the same brand as the invitation issuer and invitation mint.

Reply:
Did you mis-read this? I think it says the issuer and mint are one-to-one.  The mint is also "devoted to minting invitations" but those aren't described as one-to-one.

Reply:
Ah, sorry. Yes, the issuer and mint are one-to-one.

Reply:
Wait, it says "they"? Is it invitations or the issuer?

Reply:
Point. I blame writing on insufficient sleep. : -)

Reply:
Although in this bit, I was referring to the one-to-one between the InvitationIssuer and the invitation mint, which is one-to-one. It does apply to the later bit though, and still rephrasing.